rust/operator-binary/src/authentication/password/ldap.rs

```diff
@@ -161,8 +161,9 @@ mod tests {
     const LDAP_SEARCH_BASE: &str = "ou=users,dc=example,dc=org";
 
     fn setup_test_authenticator() -> LdapAuthenticator {
-        let auth_provider = serde_yaml::from_str::<ldap::AuthenticationProvider>(&format!(
-            r#"
+        let auth_provider =
+            serde_yaml::from_str::<ldap::v1alpha1::AuthenticationProvider>(&format!(
+                r#"
```

Comment:
I prefer the previous indentation, but that's a matter of taste

Reply:
I think for this particular line rustfmt struggles with the r#" and following lines and leaves them unchanged (did not test it). but not important